hw2/main.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask
+
+import random
+import string
+import pandas as pd
+
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello_world():
+    return "<p>Hello, World!</p>"
+
+
+@app.route("/random_password")
+def generate_password():
+    n = random.randint(10,20)
+    characters = string.ascii_letters + string.digits + string.punctuation
+    password = ''.join(random.choice(characters) for i in range(n))
+    return password
```

Comment:
Якщо змінная ніде не використовується, то ії іменують _ . Це дає родуміння іншому розробнику, що ця змінна ніде не буде використана.

```suggestion
    return ''.join(random.choice(characters) for _ in range(n))
```